Tests/SwiftParserTest/Expressions.swift

```diff
@@ -509,10 +509,10 @@ final class ExpressionTests: XCTestCase {
       let #^VAR_NAME^#:(#^DIAG_1^#..)->#^END^#
       """,
       diagnostics: [
-        DiagnosticSpec(locationMarker: "VAR_NAME", message: "Expected pattern after 'let' in variable"),
-        DiagnosticSpec(locationMarker: "DIAG_1", message: "Expected type after '(' in function type"),
+        DiagnosticSpec(locationMarker: "VAR_NAME", message: "Expected pattern in variable"),
+        DiagnosticSpec(locationMarker: "DIAG_1", message: "Expected type in function type"),
         DiagnosticSpec(locationMarker: "DIAG_1", message: "Unexpected text '..' in function type"),
-        DiagnosticSpec(locationMarker: "END", message: "Expected type after '->' in function type"),
+        DiagnosticSpec(locationMarker: "END", message: "Expected type in function type"),
```

Comment:
Similar to the above, without `after '->'` this would really need to be `expected return type`.